components/needs_team/needs_team.tsx

```diff
@@ -151,6 +168,10 @@ export default class NeedsTeam extends React.Component {
         window.removeEventListener('keydown', this.onShortcutKeyDown);
     }
 
+    shouldComponentUpdate() {
```

Comment:
Why was this added?

Reply:
There were following eslint error on this file:
Component is not optimized. Please add a shouldComponentUpdate method.eslint(react/require-optimization)

Reply:
If it's just a warning, we can safely ignore it.
Since this method isn't doing anything, we can remove it.

Reply:
Done.